API/Dockerfile

```diff
@@ -5,13 +5,16 @@ FROM golang:latest AS builder
 USER root
 WORKDIR /home
 
-COPY API/ .git* /home/
+RUN apt install git
+
+COPY API/go.mod API/go.mod /home/
```

Comment:
API/go.mod twice?

Reply:
oups, the second should be go.sum, im fixing this